assessment.js

```diff
@@ -1,8 +1,6 @@
 //We'll learn about require later in the course
-var ask = require('./ask.js');
-
-var answer = 'not empty';
-
-while (answer !== '' && answer !== 'SeCrEt') {
-  answer = ask("Guess my secret? ");
+for (var i = 1; i < 11; i++){
+  if (i === 4){
+    console.log("Four!")
+  }
```

Comment:
Not quite what we were looking for. The program should allow users to guess until they get the number, e.g.
```javascript
var answer = ''";
while (answer !== '7') {
  answer = ask("Guess a number between 1 and 10");
}
```